fe/fe-core/src/main/java/org/apache/doris/nereids/rules/expression/rules/OneListPartitionEvaluator.java

```diff
@@ -35,25 +35,25 @@
 import java.util.stream.IntStream;
 
 /** OneListPartitionInputs */
-public class OneListPartitionEvaluator
-        extends DefaultExpressionRewriter<Map<Slot, PartitionSlotInput>> implements OnePartitionEvaluator {
-    private final long partitionId;
+public class OneListPartitionEvaluator<K>
```

Comment:
Why add this <K>， this seems not used by this pr

Reply:
The ID used for partitioning the inner table, the name used for the external table, and the underlying clipping method are compatible with generics